docs/reference/cli.md

```diff
@@ -30,6 +30,7 @@ Commands:
   info                    Print information about a SQLMesh project.
   init                    Create a new SQLMesh repository.
   invalidate              Invalidate the target environment, forcing its...
+  janitor                 Run the janitor task on-demand.
```

Comment:
Should we also expose a notebook magic for this?

Reply:
Added